README.md

```diff
@@ -84,14 +84,14 @@ Our default export contains all of our Stylelint rules, along with specific plug
 | [`scale-unlimited/declaration-strict-value`][37]             | Enabled - [see Config][config]                                      |
 | [`scss/at-extend-no-missing-placeholder`][38]                | Enabled                                                             |
 | [`scss/at-if-no-null`][39]                                   | Enabled                                                             |
-| [`scss/at-import-no-partial-leading-underscore`][40]         | Enabled                                                             |
-| [`scss/at-import-partial-extension-blacklist`][41]           | `"scss"`                                                            |
-| [`scss/at-rule-no-unknown`][42]                              | Enabled                                                             |
-| [`scss/comment-no-empty`][43]                                | Enabled                                                             |
-| [`scss/declaration-nested-properties-no-divided-groups`][44] | Enabled                                                             |
-| [`scss/dollar-variable-no-missing-interpolation`][45]        | Enabled                                                             |
-| [`scss/function-quote-no-quoted-strings-inside`][46]         | Enabled                                                             |
-| [`scss/function-unquote-no-unquoted-strings-inside`][47]     | Enabled                                                             |
+| [`scss/at-import-partial-extension-blacklist`][40]           | `"scss"`                                                            |
+| [`scss/at-rule-no-unknown`][41]                              | Enabled                                                             |
+| [`scss/comment-no-empty`][42]                                | Enabled                                                             |
+| [`scss/declaration-nested-properties-no-divided-groups`][43] | Enabled                                                             |
+| [`scss/dollar-variable-no-missing-interpolation`][44]        | Enabled                                                             |
+| [`scss/function-quote-no-quoted-strings-inside`][45]         | Enabled                                                             |
+| [`scss/function-unquote-no-unquoted-strings-inside`][46]     | Enabled                                                             |
+| [`scss/load-no-partial-leading-underscore`][47]              | Enabled                                                             |
```

Comment:
I think it would produce a much smaller diff if we just update `scss/at-import-no-partial-leading-underscore` with `scss/load-no-partial-leading-underscore` instead of removing the former and appending the latter to the bottom

Reply:
This listing is generated automatically. It would probably be a lot more work to make it ordered custom.

Reply:
See https://github.com/wagtail/stylelint-config-wagtail/pull/27 for context.

Reply:
Oh, I didn't know that. That's cool, thanks!